public/media_engagement/ahadi_media_engagement.html

```diff
@@ -17,21 +22,40 @@
     <script type="text/javascript" src="firebase.js"></script>
 </head>
 <body>
-    <nav class="nav">
-        <div class="nav-wrapper container">
-            <img class='brand-logo' src="avfdashboardlogo.png">
-            <ul class="right">
-                <li><a class="nav-link" href="https://avf-dashboards.firebaseapp.com/coda/coding_progress.html">MESSAGE CODING</a></li>
-                <li><a class="nav-link" href="https://avf-dashboards.firebaseapp.com/media_engagement/ahadi_media_engagement.html">AHADI_E&M</a></li>
-                <li><a class="nav-link" href="https://avf-dashboards.firebaseapp.com/media_engagement/icraf_media_engagement.html">ICRAF_E&M</a></li>
-                <li><a class="nav-link" href="https://avf-dashboards.firebaseapp.com/media_engagement/imaqal_media_engagement.html">IMAQAL_E&M</a></li>
-                <li><a class="nav-link" href="https://avf-dashboards.firebaseapp.com/media_engagement/ocha_media_engagement.html">OCHA_E&M</a></li>
-                <li><a class="nav-link" href="https://avf-dashboards.firebaseapp.com/media_engagement/undp_rco_media_engagement.html">UNDP_RCO_E&M</a></li>
-                <li><a class="nav-link" onClick="logout();">Logout</a></li>
-            </ul>
+    <!-- NAVBAR WITH DROPDOWN & RESPONSIVE TOGGLE-->
+    <nav class="navbar navbar-expand-sm navbar-brown bg-white mb-3">
+        <div class="container">
+            <a class="navbar-brand" href="#"><img class="img-fluid logo" src="avfdashboardlogo.PNG" alt="avf-logo"></a>
+
+            <!-- TOOGLE TRIGGER FOR SMALL DEVICES -->
+            <button class="navbar-light navbar-toggler" data-toggle="collapse" data-target="#navbarNav">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <!-- RESPONSIVE TOGGLE -->
+            <div class="collapse navbar-collapse" id="navbarNav">
+                <ul class="navbar-nav ml-auto">
+                    <li><a class="nav-link border-left border-top border-right" href="coda/coding_progress.html">MESSAGE CODING</a></li>
+
+                    <li class="nav-item dropdown">
+                        <!-- DROPDOWN TRIGGER -->
+                        <a href="#" class="nav-link border-top border-right active-link dropdown-toggle" data-toggle="dropdown">MEDIA ENGAGEMENT</a> 
```

Comment:
We don't call this 'media and engagement' anymore its abbreviation is the same as 'monitoring and evaluation'. Please revert to either 'engagement and media', or, even better, 'traffic', as these pages do not currently report any engagement sats